dashboard/app/views/levels/_external.html.haml

```diff
@@ -1,6 +1,7 @@
 %link{rel: 'stylesheet', href: asset_path('blockly/css/common.css')}
 = stylesheet_link_tag 'levels/external', media: 'all'
 = javascript_include_tag 'levels/external'
+-# TODO(dave): Remove options global once all templates consume externalOptions via erb.
```

Comment:
While I agree it's not great to hang `options` on global `window`, I don't think having each partial implement rendering of `externalOptions` is necessarily a good thing either.  Ideally this would be somehow sent through `appOptions` since that's our primary collection for passing data from Dashboard to the client.